proto/aquila/action_execute.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+
+option ruby_package = "Tucana::Aquila";
+
+package aquila;
+
+message ActionCallRequest {
+  repeated string parameters = 1;
+}
+
+message ActionCallResponse {
+  string execution_identifier = 1;
+}
+
+message ActionResult {
+  repeated string result = 1;
+}
+
+message ActionResultRequest {}
```

Comment:
Execution sends this message to get optional results from Aquila

Reply:
Missing `execution_identifier ` to get the linked result and not just any